core/trino-main/src/test/java/io/trino/operator/aggregation/listagg/TestListaggAggregationFunction.java

```diff
@@ -82,7 +82,7 @@ public void testInputEmptyState()
     @Test
     public void testInputOverflowOverflowFillerTooLong()
     {
-        String overflowFillerTooLong = StringUtils.repeat(".", 65_537);
```

Comment:
These changes will be needed for Airbase 127. I just noticed they didn't get into 126.